tests/chart_tests/test_houston_configmap.py

```diff
@@ -175,10 +182,15 @@ def test_houston_configmap_with_loggingsidecar_enabled_with_overrides():
     """Validate the houston configmap and its embedded data with loggingSidecar."""
     sidecar_container_name = "sidecar-log-test"
     terminationEndpoint = "http://localhost:8000/quitquitquit"
+    image_name = "quay.io/astronomer/ap-vector:0.22.3"
```

Comment:
should this be 0.23.0?

Reply:
this is a static test case @vishwas-astro